### ISSUE TITLE
Cannot load a read-only chains.json file

### ISSUE THREAD

**Issue by lander2k2** [pokt-network/pocket-core#1391]
**Describe the bug**
Cannot open a chains.json file that is on a read-only filesystem.

I'm running a pocket node on Kubernetes with the `chains.json` file in a read-only volume mounted from a K8s secret since there are relay chain node passwords in there.

Error:
```
2022/02/08 21:26:24 invalid chains.json: open /root/.pocket/config/chains/chains.json: read-only file system
```

**To Reproduce**
1. Put `chains.json` on a read-only file system
2. Update `pocket_config.chains_name` in `config.json` to point to the correct location for the chains file
3. Run `pocket start`

**Expected behavior**
I expect to be able to load a read-only chains file.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - Linux

**Smartphone (please complete the following information):**
N/A

**Additional context**
It's a one-line fix in the code.  I forked and tested.  Will submit a PR.

**Comment by lander2k2** [comment]
This problem also applies to the config.json file.  When mounting this to a Kubernetes pod with a configmap it fails to load for the same reason: read only filesystem.

**Comment by oten91** [comment]
Merged on RC-0.8.3
